kinto_amo/views/services.py

```diff
@@ -9,13 +12,58 @@
 
 @blocklist.get()
```

Comment:
Since we define our own view here, it maybe relevant to use a renderer instead, what do you think ?